app/discovery/provider/docker_test.go

```diff
@@ -86,97 +69,88 @@ func TestDocker_List(t *testing.T) {
 	assert.Equal(t, "*", res[2].Server)
 }
 
-func TestDocker_ListWithAutoAPI(t *testing.T) {
-	dclient := &DockerClientMock{
-		ListContainersFunc: func(opts dc.ListContainersOptions) ([]dc.APIContainers, error) {
-			return []dc.APIContainers{
-				{Names: []string{"c1"}, State: "running",
-					Networks: dc.NetworkList{
-						Networks: map[string]dc.ContainerNetwork{"bridge": {IPAddress: "127.0.0.2"}},
-					},
-					Ports: []dc.APIPort{
-						{PrivatePort: 1345},
-						{PrivatePort: 12345},
-					},
-					Labels: map[string]string{"reproxy.route": "^/api/123/(.*)", "reproxy.dest": "/blah/$1",
-						"reproxy.port": "12345", "reproxy.server": "example.com, example2.com", "reproxy.ping": "/ping"},
-				},
-				{Names: []string{"c2"}, State: "running",
-					Networks: dc.NetworkList{
-						Networks: map[string]dc.ContainerNetwork{"bridge": {IPAddress: "127.0.0.3"}},
-					},
-					Ports: []dc.APIPort{
-						{PrivatePort: 12346},
-					},
-				},
-				{Names: []string{"c3"}, State: "stopped"},
-				{Names: []string{"c4"}, State: "running",
-					Networks: dc.NetworkList{
-						Networks: map[string]dc.ContainerNetwork{"other": {IPAddress: "127.0.0.2"}},
-					},
-					Ports: []dc.APIPort{
-						{PrivatePort: 12345},
-					},
-				},
-				{Names: []string{"c5"}, State: "running",
-					Networks: dc.NetworkList{
-						Networks: map[string]dc.ContainerNetwork{"bridge": {IPAddress: "127.0.0.122"}},
-					},
-					Ports: []dc.APIPort{
-						{PrivatePort: 2345},
-					},
-					Labels: map[string]string{"reproxy.enabled": "false"},
-				},
-			}, nil
+func TestDocker_refresh(t *testing.T) {
```

Comment:
Is there a reason you removed ListWithAutoAPI test? It has very little to do with the underlying docker client implementation and intended to make sure the label-less containers properly handled with auth mode

Reply:
Oops. Forgot about it.